Amazon_Payment/web/template/form/element/email.html

```diff
@@ -51,32 +51,90 @@
             class="fieldset hidden-fields"
             data-bind="fadeVisible: isPasswordVisible"
         >
-            <div class="field">
+            <div
+                class="
+                    field
+                    input
+                    input--password
+                "
+            >
                 <label
-                    class="label"
+                    class="input__label"
                     for="customer-password"
                 >
                     <span data-bind="i18n: 'Password'"></span>
                 </label>
                 <div class="control">
                     <input
-                        class="input-text"
+                        class="
+                            input-text
+                            input__field
+                        "
                         placeholder="optional"
                         type="password"
                         name="password"
                         id="customer-password"
-                        data-validate="{required:true}"
+                        data-validate="{ required: true }"
                         autocomplete="off"
                     />
+                    <button
+                        type="button"
+                        class="
+                            button
+                            button--icon
+                            input__button-pass
+                        "
+                        data-bind='
+                            mageInit: { "passwordTrigger": {} },
+                            attr: {
+                                "aria-label": $t("Show/hide password")
+                            }
+                        '
+                    >
+                        <svg
+                            class="
+                                icon
+                                icon--secondary
+                                input__button-pass-icon-view
+                            "
+                            role="presentation"
+                            focusable="false"
+                        >
+                            <title data-bind="i18n: 'View password icon'"></title>
```

Comment:
can be removed if role="presentation" and focusable="false"

Reply:
👍  done